.scalafmt.conf

```diff
@@ -1,5 +1,5 @@
 version = "3.8.1"
-runner.dialect = Scala213Source3
```

Comment:
Is this the core of this PR or could/should be actually moved to another PR including some discussion around the dialect topic? This is the also most probably the reason some formatting failed on some files. I'm also not sure if this change will trigger more formatting noise in the upcoming PRs. This is also most probably the reason of grouping combines classes mentioned in another comment. 
At the end we would like start to use Scala 3 in this repo, but the Scala 2 legacy code will be still here, so I'm not sure if we should change just like that. What do you think?

Reply:
`Scala213Source3` to `Scala3` does not do anything to our formatting since we already used the Scala 3 syntax.

In `.scalafix.conf` I have left the import on `Scala 2` syntax so that the wildcard imports are still `_` instead of `*`. Will change that in a followup, this PR is already too noise.

Reply:
Is `Scala3` the only choice for Scala 3?

Reply:
@siers all available dialects are listed in the link above. There is really not much info about them but it seems so there is only one `scala3` related. I guess things could be adjusted with more in detail settings and you can also override dialects for particular paths, but I'm not sure the second is the way to go.

Reply:
Although, @mpro7, as Christian says it only changes one line in formatting if we set it back to Scala213Source3. Can it stay as it is then?

Reply:
The target dialect in `scalafix.conf` will be changed in https://github.com/dasch-swiss/dsp-api/pull/3236/files